indykite/resource_ingest_pipeline_test.go

```diff
@@ -161,7 +168,15 @@ var _ = Describe("Resource IngestPipeline", func() {
 				)})),
 				"Config": PointTo(MatchFields(IgnoreExtras, Fields{
 					"IngestPipelineConfig": test.EqualProto(
-						expectedUpdatedResp.GetConfigNode().GetIngestPipelineConfig(),
+						&configpb.IngestPipelineConfig{
+							Sources: []string{"source1", "source2", "source3"},
+							Operations: []configpb.IngestPipelineOperation{
+								configpb.IngestPipelineOperation_INGEST_PIPELINE_OPERATION_UPSERT_NODE,
+								configpb.IngestPipelineOperation_INGEST_PIPELINE_OPERATION_UPSERT_RELATIONSHIP,
+								configpb.IngestPipelineOperation_INGEST_PIPELINE_OPERATION_DELETE_NODE,
```

Comment:
Why this was replaced from `expectedUpdatedResp`? I think `expectedUpdatedResp` is valid in this case.

Reply:
Because after the last fix which [has an empty string AppAgentToken](https://github.com/indykite/terraform-provider-indykite/pull/83/files#diff-83960a8bf14c72a337ad7238668cb1558fde76a22b6d7fa5532061ed4688eff2L104-R104) we cannot use the object directly, as when updating or creating we are passing a value for the token